### PR TITLE
lint: Add ESLint for TriggerSet Property Order

### PR DESCRIPTION
**Pull request by panicstevenson**

### CODE REVIEW

**Comment by panicstevenson** [comment]
Mentioned in https://github.com/quisquous/cactbot/pull/3099

**Comment by quisquous** [comment]
Is everything really already in order?!?!

**Comment by panicstevenson** [comment]
> Is everything really already in order?!?!

This is only checking the TypeScript raidboss files, so the sample size isn’t very large.

File: eslint/cactbot-triggerset-property-order.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      recommended: true,
+      url: 'https://github.com/quisquous/cactbot/blob/main/docs/RaidbossGuide.md#trigger-elements',
+    },
+    fixable: 'code',
+    messages: {
+      sortKeys: 'Expected triggerSet properties ordered like {{expectedOrder}} (\'{{beforeKey}}\' should be before \'{{nextKey}}\')',
+    },
+  },
+  create: (context) => {
+    const orderList = [
```
Comment (quisquous) [inline eslint/cactbot-triggerset-property-order.js:18]: Could you do the same oopsy/raidboss module dance here?

File: .eslintrc.cjs [hunk 2]
```diff
@@ -268,9 +268,10 @@ const overrides = [
     },
   },
   {
-    'files': ['**/oopsyraidsy/data/**/*.ts', '**/raidboss/data/**/*.ts'],
+    'files': ['**/raidboss/data/**/*.ts'],
     'rules': {
       'rulesdir/cactbot-matches-type-references': 'error',
+      'rulesdir/cactbot-triggerset-property-order': ['error', { 'module': 'raidboss' }],
```
Comment (quisquous) [inline .eslintrc.cjs:274]: Can you add this equivalent to the new oopsy-only section too?

**Pull request merged**